src/embedded.test.js

```diff
@@ -698,12 +684,12 @@ describe('HelloSign', () => {
       });
 
       test('emits the "message" event when the app has posted a message', (done) => {
-        client = new HelloSign({ clientId: mockClientId });
+        client = new HelloSign({ clientId: mockGuid });
 
         const message = {
           type: HelloSign.messages.APP_INITIALIZE,
           payload: {
-            signatureId: mockSignatureId,
+            signatureId: mockGuid,
```

Comment:
is this correct? we are replacing both mockClientId and mockSignatureId with the same value?

Reply:
Yeah, these all used to be separate (see [here](https://github.com/hellosign/hellosign-embedded/pull/164/files#diff-74052aaa7c4f5d29bd86f6bde5c300f5L6-L8)), but when I was in this file messing with the tests, I decided that there was no need to have three variables, and just combined them into a single `mockGuid`